.travis.yml

```diff
@@ -57,20 +58,22 @@ virtualenv:
 after_success:
 - |
   if [[ "$TRAVIS_TAG" == v* ]]; then
+    LANDLAB_PACKAGE=$(conda build --output ./.conda-recipe --old-build-string)
+    anaconda -t $ANACONDA_TOKEN upload --force --user landlab --channel main $LANDLAB_PACKAGE
     bash .ci/travis/deploy_to_pypi.sh
-    bash .ci/travis/deploy_to_anaconda.sh
+    # bash .ci/travis/deploy_to_anaconda.sh
   fi
 - if [[ "$COVERALLS" == "1" ]]; then coveralls; fi
-deploy:
-  provider: pypi
-  user: mcflugen
-  password:
-    secure: V15DceMv4DUmjgxToaxrP3KaQxsLbfrTkB1s6D8YNQZvYJiVvNlBpC1nF8yWbOt3IWJQSGjaQEnObd2ZhHe2Kq/H+QsEFyBKlYF7SNp35FyS5zLuf2qhnNhmzhqkspwBpiGe88c1PQPFzfqdIUQXp7ZAv6L8YRvNy2sbzvgxh6I=
-  on:
-    tags: true
-    branch: release
-    repo: landlab/landlab
-  distributions: sdist bdist_wheel
+# deploy:
```

Comment:
@mcflugen just checking that you meant to keep all of the deploy portion commented out. 
assuming so, this looks oK to me.

Reply:
Yes, that is fine. We deploy to both Anaconda Cloud and PyPI in the `after_success` step so this isn't necessary.